gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ddl/catalog/GfxdSystemProcedures.java

```diff
@@ -2462,6 +2463,7 @@ public static void SET_NANOTIMER_TYPE(Boolean useNativeTimer, String nativeTimer
         GfxdSystemProcedureMessage.SysProcMethod.setNanoTimerType, false, true);
   }
 
+  // TODO: send to all nodes after doing local commit
```

Comment:
Is this still TODO? If this is required, how this is working without this. Same question for rollback as well.